contracts/contracts/lib/ECVerify.sol

```diff
@@ -2,51 +2,36 @@ pragma solidity ^0.4.17;
 
 library ECVerify {
 
-    function safer_ecrecover(bytes32 hash, uint8 v, bytes32 r, bytes32 s) internal returns (bool, address) {
-        bool ret;
-        address addr;
+    function ecverify(bytes32 hash, bytes signature) internal pure returns (address) {
```

Comment:
As discussed, please use `(address addr)` as return type for addr, should work as well.